aws/ecs/cluster/main.tf

```
@@ -3,6 +3,19 @@
  */
 resource "aws_ecs_cluster" "ecs_cluster" {
   name = var.cluster_name == "" ? "${var.app_name}-${var.app_env}" : var.cluster_name
+
+  tags = merge({
+    app_name = var.app_name
+    app_env  = var.app_env
+  }, var.tags)
```

```suggestion
  tags = var.tags
```
It would be better, I think, to not add these tags. We don't use `app_name` and `app_env` tags now because the namespace was polluted in a sense. Some `app_env` values were abbreviated and some were not, so we moved to a different tag name to get away from this issue. Regardless, being an open source library, it's better to leave configuration to the consumer whenever practical.

---

Depending on why you added this, it may not even be necessary. We have been using the `default_tags` on the `aws` provider to avoid having to add this to every resource and module individually.

---

Thanks.  The provider default tags will help.
Although this module still wants to remove the "AWS.SSM.AppManager.ECS.Cluster.ARN" tag without this change.

---

Do you still want this `tags = var.tags` line even with provider default tags?